commands/pull.js

```diff
@@ -0,0 +1,47 @@
+const cli = require('heroku-cli-util')
+const Sanbashi = require('../lib/sanbashi')
+const debug = require('../lib/debug')
+
+let usage = `
+    ${cli.color.bold.underline.magenta('Usage:')}
+    ${cli.color.cmd('heroku container:pull web')}        # Pulls the web image from the app
+    ${cli.color.cmd('heroku container:pull web worker')} # Pulls both the web and worker images from the app
+    ${cli.color.cmd('heroku container:push web:latest')} # Pulls the latest tag from the web image`
```

Comment:
Do we want to advertise how to use tags? It's currently unspecified how they work with the registry.

Reply:
I've set it based on the current RFC we have. But we can add them later based on how we end up doing the rest of the work.
@jbyrum what do you think?

Reply:
Looks like the only way is to make an HTTP call to the registry: https://stackoverflow.com/a/31750543/122080
We could make that a `container:list` command.

Reply:
Just closing the loop on this -- we discussed in our team meeting to not support pulling tags at this time

Reply:
Also, the third help command says `push` when it should say `heroku container:pull`